platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/impl/GradleEnterprisePluginAutoAppliedStatus.java

```diff
@@ -16,6 +16,10 @@
 
 package org.gradle.internal.enterprise.impl;
 
+import org.gradle.internal.service.scopes.Scopes;
+import org.gradle.internal.service.scopes.ServiceScope;
+
+@ServiceScope(Scopes.BuildTree.class)
```

Comment:
❓ How can the status be `BuildTree` while the listener is of scope`Build`? I don't know if we have an auto application per build tree or per build, though auto-application per build tree would make sense to me.

Reply:
The annotation was removed after audit